src/AppBundle/Resources/views/Report/Gift/_form.html.twig

```diff
@@ -10,6 +12,12 @@
         {{ form_select(form.bankAccountId, 'form.bankAccount') }}
     {% endif %}
 
-    {{ macros.saveAndContinueButton(form.save) }}
+    {{ macros.saveAndContinueButton(form.saveAndContinue) }}
+
+    {% if form.saveAndAddAnother is defined %}
+        <span class="push-half--sides text--center">or</span>
```

Comment:
This is using old GOV.UK Elements classes. I believe the classes should be `govuk-!-margin-left-3 govuk-!-margin-right-3`. I don't think `text--center` would do anything useful in this case.

Reply:
Sorry, correction: This is using some custom DigiDeps classes which are now part of the Design System, so we should use the Design System versions instead. `govuk-!-margin-left-3 govuk-!-margin-right-3` should do it.

Reply:
will fix. Ta

Reply:
Fixed, applied gov uk style classes to macros and removed span altogether